### ISSUE TITLE
app.use('qt5') Press ctrl or ctrl + command error

### ISSUE THREAD

**Issue by lminggang** [glumpy/glumpy#234]
1. Mac uses virtual machine Linux system, add the code app.use('qt5') to the brain.py file window, press ctrl code 338 error, press ctrl + command code 342 error, it is recommended that the backend_qt5.py code be updated to backend_glfw.py 330 -341 lines of code.

2. Backend_qt5.py 321 - 343 Code duplication

**Comment by rougier** [comment]
I'm sorry but I'm not sure I understand the error and the code duplication you're referring to. Could you give some more details (and log output) ?

**Comment by erte0011** [comment]
TL;DR change 'MOD_CONTROL' to 'MOD_CTRL' in backend_qt5.py

Apologies if I am doing this wrong, it is my first post. I am having the same issue and I believe the fix Iminggang is referring to is as follows...

How to reproduce; use qt5 as the backend, press the control key and the following error occurs;

File ".\glumpy\app\window\backends\backend_qt5.py", line 291, in key_press_event
  modifiers = self._modifiers_translate(event.modifiers())
File ".\glumpy\app\window\backends\backend_qt5.py", line 331, in _modifiers_translate
  _modifiers |= window.key.MOD_CONTROL
AttributeError: module 'glumpy.app.window.key' has no attribute 'MOD_CONTROL'

How to fix; in backend_qt5.py line 331 change MOD_CONTROL to MOD_CTRL.

Also note there is a duplicate '_modifiers_translate' method in backend_qt5.py.

**Comment by rougier** [comment]
Oh, I see now. Can you make a PR to fix the problem?
